packages/internal/bridge/sdk/src/lib/tenderly.ts

```diff
@@ -117,8 +121,52 @@ export async function submitTenderlySimulations(
         BridgeErrorType.TENDERLY_GAS_ESTIMATE_FAILED,
       );
     }
-    gas.push(simResults[i].gasUsed);
+    if (simResults[i].logs === undefined) {
+      throw new BridgeError(
+        'Estimating gas did not return logs',
+        BridgeErrorType.TENDERLY_GAS_ESTIMATE_FAILED,
+      );
+    }
+    // Attempt to extract event.
+    for (let j = 0; j < simResults[i].logs.length; j++) {
```

Comment:
would this be unnecessary work when estimates are for deposits? if so, should this be moved out from this function and performed by the calling fn, which would have the right context.